hyprbars | issue during compilation

when executing 'make all' for hyprbars, i get this issue:
```
In file included from /usr/local/include/hyprland/src/includes.hpp:53,
                 from /usr/local/include/hyprland/src/defines.hpp:1,
                 from /usr/local/include/hyprland/src/Compositor.hpp:7,
                 from main.cpp:6:
/usr/local/include/hyprland/wlroots/wlr/types/wlr_layer_shell_v1.h:16:10: fatal error: wlr-layer-shell-unstable-v1-protocol.h: No such file or directory
   16 | #include "wlr-layer-shell-unstable-v1-protocol.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /usr/local/include/hyprland/src/includes.hpp:53,
                 from /usr/local/include/hyprland/src/defines.hpp:1,
                 from /usr/local/include/hyprland/src/render/decorations/IHyprWindowDecoration.hpp:3,
                 from barDeco.hpp:5,
                 from barDeco.cpp:1:
/usr/local/include/hyprland/wlroots/wlr/types/wlr_layer_shell_v1.h:16:10: fatal error: wlr-layer-shell-unstable-v1-protocol.h: No such file or directory
   16 | #include "wlr-layer-shell-unstable-v1-protocol.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:12: hyprbars.so] Error 1
```

hyprctl version: 
```
Hyprland, built from branch  at commit f27873a6f06dc2f87600edb890f3c38298bfb55f dirty ().
Tag: v0.25.0
```

--- a/hyprbars/Makefile
+++ b/hyprbars/Makefile
@@ -1,6 +1,6 @@
 CXX = g++
 CXXFLAGS = -shared -fPIC --no-gnu-unique -g -std=c++23
-INCLUDES = -I "/usr/include/pixman-1" -I "/usr/include/libdrm" -I $(shell pkg-config --cflags hyprland pangocairo)
+INCLUDES = -I "/usr/include/pixman-1" -I "/usr/include/libdrm" $(shell pkg-config --cflags hyprland pangocairo)
 LIBS = $(shell pkg-config --libs pangocairo)
 
 SRC = main.cpp barDeco.cpp

